1Homework18.02.22/SquareSorting/SquareSorting/Program.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+
+namespace SquareSorting
+{
+    internal class Program
+    {
+        /// <summary>
+        /// Checks if an array is sorted
+        /// </summary>
+        /// <param name="array"></param>
+        /// <returns></returns>
+        private static bool AreArraySorted(int[] array)
+        {
+            for (int i = 0; i < array.Length - 1; ++i)
+            {
+                if (array[i] > array[i + 1])
+                {
+                    return false;
+                }
+            }
+
+            return true;
+        }
+        /// <summary>
```

Comment:
```suggestion
        }
        
        /// <summary>
```
Методы разделяются пустой строкой всегда